aten/src/ATen/native/TensorShape.cpp

```diff
@@ -813,6 +813,11 @@ Tensor diag_embed(const Tensor& self, int64_t offset, int64_t dim1_, int64_t dim
   return result;
 }
 
+Tensor expand_symint(const Tensor& self, c10::SymIntArrayRef packed_size, bool /*unused*/) {
+  auto size = ExpectIntArrayRef(packed_size);
+  return expand(self, size, false);
```

Comment:
Hmm? How come you don't just propagate the boolean through?

Reply:
oh crap :(